apps/frontend/site/.env.example

```
@@ -18,6 +18,7 @@ PUBLIC_DISABLE_APP_CHECK=
 PUBLIC_MODE=emulator
 PUBLIC_LOG_PERSIST_LEVEL=
 PUBLIC_APP_ID=site
+PUBLIC_SITE_URL=
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The PUBLIC_SITE_URL key is duplicated

(DuplicatedKey)